fecgov/openFEC
# Feature/4112 schedule h4 endpoint

## Summary (required)

- Resolves #4112 
- This is a basic first iteration working version of a new `schedule_h4` endpiont. Subsequent issues will be iterated over to refine this endpoint and possibly generalize to schedule H (rather than just H4). Additional schedule_h4 specific endpoints may also be generated using this branch as a base.

## How to test the changes locally
- checkout out branch
- pytest

To test the published version:
- uncomment the line in `rest.py`: `# apidoc.register(sched_h4.ScheduleH4View, blueprint='v1')`
- run local api, point to a db having schedule h4 data, and inspect the schedule h4 endpoint under the "disbursements" tag to see that it functions.

## Impacted areas of the application
disbursements api endpoint
